### PR TITLE
Updates for curly brackets - Pt. 1

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/roku/initial_sdk_setup.md [hunk 1]
```diff
@@ -35,9 +35,9 @@ Within `main.brs`, set the Braze configuration on the global node:
 globalNode = screen.getGlobalNode()
 config = {}
 config_fields = BrazeConstants().BRAZE_CONFIG_FIELDS
-config[config_fields.API_KEY] = "YOUR_API_KEY_HERE"
+config[config_fields.API_KEY] = "YOUR_API_KEY"
```
Comment (rachel-feinberg) [inline _docs/_developer_guide/platform_integration_guides/roku/initial_sdk_setup.md:38]: Should this be "{YOUR_API_KEY}"?

File: _docs/_developer_guide/platform_integration_guides/roku/initial_sdk_setup.md [hunk 2]
```diff
@@ -35,9 +35,9 @@ Within `main.brs`, set the Braze configuration on the global node:
...
 ' example endpoint: "https://sdk.iad-01.braze.com/"
-config[config_fields.ENDPOINT] = "YOUR_ENDPOINT_HERE"
+config[config_fields.ENDPOINT] = "YOUR_ENDPOINT"
```
Comment (rachel-feinberg) [inline _docs/_developer_guide/platform_integration_guides/roku/initial_sdk_setup.md:40]: Should this be "{YOUR_ENDPOINT}"? (include both brackets and quotes"

**Review by rachel-feinberg — APPROVED** [review]
Lgtm! Just questions for clarification. (I'm figuring out when to use brackets vs. just quotes)

**Comment by lydia-xie** [comment]
> 

Fixed it! Should be curly brackets :)

**Pull request merged**